service/raft/group0_state_machine.cc

```diff
@@ -325,7 +325,7 @@ future<> group0_state_machine::transfer_snapshot(raft::server_id from_id, raft::
             return {std::move(topology_snp), std::move(raft_snp)};
```

Comment:
There is no test. Missing `git add`?

Reply:
Oops - I indeed forgot. Will add it in the next version.